lib/classes/error-message.js

```diff
@@ -67,7 +67,7 @@ var isObject = lodash.isObject;
 function ErrorMessage() {
 
   this.eventTime = (new Date()).toISOString();
-  this.serviceContext = {service : '', version : ''};
+  this.serviceContext = {service : 'node', version : undefined};
```

Comment:
Is it preferable to define `version` to the value `undefined` or would it be better to omit the key `version` from serviceContext altogether? What does the errors API prefer?